.gitmodules

```diff
@@ -1,3 +1,6 @@
-[submodule "submodules/kuksa-databroker"]
+
+
+[submodule "kuksa-databroker"]
 	path = submodules/kuksa-databroker
-	url = https://github.com/eclipse-kuksa/kuksa-databroker
+	url = https://github.com/boschglobal/kuksa-databroker.git
```

Comment:
yes, temporary solution since we do not have v2 yet

Reply:
But we can use https://github.com/eclipse-kuksa/kuksa-databroker/tree/feature/databroker-api-v2 already now, right?

And we anyway do not want to merge this one into main until Databroker supports that on main, or?

Reply:
If it is backward compatible I am fine with merging, but better replace url and branch to point at the current feature branch in eclipse-kuksas (instead of boschglobal)